mocks/mock-bip-claims-api/src/main/java/gov/va/vro/mockbipclaims/controller/BaseController.java

```diff
@@ -0,0 +1,46 @@
+package gov.va.vro.mockbipclaims.controller;
+
+import gov.va.vro.mockbipclaims.model.bip.Message;
+import gov.va.vro.mockbipclaims.model.bip.ProviderResponse;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.time.OffsetDateTime;
+
+public class BaseController {
+  protected static final int CLAIM_YIELDS_500 = 500;
+
+  protected <T extends ProviderResponse> ResponseEntity<T> create200(T response) {
```

Comment:
Is there value to merging the methods into a single method with a switch case on a status parameter that's passed in from the calling method? Such as changing to a createResponse(response, httpstatus) method instead.